examples/make_datapipe.py

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
These are great examples but maybe we should move them to modulus-launch?

Reply:
Yeah I will remove them for now. I was just testing it.